src/Game.ts

```diff
@@ -827,6 +848,20 @@ export class Game extends EventEmitter<Events> {
     }
 }
 
+export function handleChatLine(game_id: string, line: any, cutoff_timestamp: number) {
+    if (typeof line.body !== "string") {
+        return;
+    }
+    if (line.username === config.username) {
+        return;
+    }
+    if (line.date < cutoff_timestamp) {
```

Comment:
dunno if it's worth noting that converting `new Date()` to a number gives you epoch time in UTC and `line.date` is also epoch time in UTC

Reply:
If you felt like asking, I'm happy to add a note!